R/http.R

```diff
@@ -131,12 +146,13 @@ POST_JSON <- function(service,
                       query = NULL,
                       headers = list()) {
   POST(service,
-       authInfo,
-       path,
-       query,
-       "application/json",
-       content = toJSON(json, pretty = TRUE, digits = 30),
-       headers = headers)
+    authInfo,
```

Comment:
nit: should `service` also be on its own line? i guess the same question applies to the function definitions. any time we end up with a series of enumerated, newline-separated arguments, the first should also be on its own line, not with the opening paren.

i find myself doing this for any nontrivial function calls and definitions, like the `POST` call here or its surrounding `POST_JSON`.

on the other hand, i find myself avoiding many-line argument lists for `cat` and `sprintf`. that's no rule, though. :)

Reply:
Yeah, agreed. I think this is a remnant of an older style that styler supports that we no longer like.